python/lib/communication/dmod/communication/_version.py

```diff
@@ -1 +1 @@
-__version__ = '0.2.1'
+__version__ = '0.2.2'
```

Comment:
are the communication additions enough to warrant a minor rev `0.3.0`?

Reply:
Yeah, will change.  See above for monitor package.